### PR TITLE
Changes to support runs on minikube

### PR DESCRIPTION
**Pull request by kusumachalasani**
Signed-off-by: kusumachalasani <kchalasa@redhat.com>

### CODE REVIEW

**Comment by chandrams** [comment]
@kusumachalasani - Can you please include a separate script for running only the load. Update the doc with steps to run the deploy & load.

File: techempower/scripts/tfb-cleanup.sh [hunk 1]
```diff
@@ -53,31 +53,37 @@ fi
...
 	#Delete the services and routes if any
-	TFB_SERVICES=($(oc get svc --namespace=${NAMESPACE} | grep -e "tfb-qrh" -e "tfb-database" | cut -d " " -f1))
+	TFB_SERVICES=($(${K_EXEC} get svc --namespace=${NAMESPACE} | grep -e "tfb-qrh" -e "tfb-database" | cut -d " " -f1))
 	for se in "${TFB_SERVICES[@]}"
 	do
-		oc delete svc ${se} --namespace=${NAMESPACE}
+		${K_EXEC} delete svc ${se} --namespace=${NAMESPACE}
 	done
-	TFB_ROUTES=($(oc get route --namespace=${NAMESPACE} | grep -e "tfb-qrh" -e "tfb-database" | cut -d " " -f1))
+	TFB_ROUTES=($(${K_EXEC} get route --namespace=${NAMESPACE} | grep -e "tfb-qrh" -e "tfb-database" | cut -d " " -f1))
```
Comment (chandrams) [inline techempower/scripts/tfb-cleanup.sh:69]: I think route is specific to openshift please add a cluster check here as the script fails with "error: the server doesn't have a resource type "route" on minikube

File: techempower/scripts/tfb-qrh-deploy.sh [hunk 2]
```diff
@@ -24,7 +24,7 @@ source ${CURRENT_DIR}/tfb-common.sh
 # SCRIPT BENCHMARK_SERVER 
 # Ex of ARGS :  -s example.in.com -i 2 -g kruize/tfb-qrh:1.13.2.F_mm.v1
 
-CLUSTER_TYPE="openshift"
+#CLUSTER_TYPE="openshift"
```
Comment (chandrams) [inline techempower/scripts/tfb-qrh-deploy.sh:27]: Remove this commented line

File: techempower/README.md [hunk 3]
```diff
@@ -1,10 +1,12 @@
 # Instructions to run the TechEmpower Framework (Quarkus) application using scripts 
-**The scripts written supports**
-- [Openshift](#Openshift)
+**The scripts written supports openshift and minikube**
```
Comment (chandrams) [inline techempower/README.md:2]: Update README based on latest changes - load script, deploy script name change etc

File: techempower/scripts/tfb-load.sh [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+                echo
+                echo "#########################################################################################"
+                echo "                             Running with Connections ${connection}                      "
+                echo "#########################################################################################"
+                echo
+
+               # Run the wrk load
+                echo "Running wrk load for ${APP_NAME} instance ${inst} with the following parameters"
+		if [ "${CLUSTER_TYPE}" == "openshift" ]; then
+			cmd="${HYPERFOIL_DIR}/wrk.sh --latency --threads=${THREADS} --connections=${CONNECTION} --duration=${DURATION}s http://${IP_ADDR}/${END_POINT}"
```
Comment (chandrams) [inline techempower/scripts/tfb-load.sh:107]: HYPERFOIL_DIR is not specified in the usage. Please document how to install hyperfoil in the readme.

Comment (chandrams) [reply techempower/scripts/tfb-load.sh:107]: I see load_setup in the common script so I think calling it here is missing.

**Review by dinogun — COMMENTED** [review]
@kusumachalasani Is this ready for review again ? @chandrams are you ok with the current changes?

**Comment by kusumachalasani** [comment]
> @kusumachalasani Is this ready for review again ?
Yes, it is ready to review.

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**